operations/enrich_results.yml

```diff
@@ -0,0 +1,31 @@
+id: enrich_results
+name: enrich_results
+description: Create new results by applying enrichment analysis to existing results.  In particular, combines results by transforming a qnode into a set, formed of knodes that share a property or relation more often than expected by chance.
+unique: true
+examples:
+- input: enrich/messages/01_input_enrich_results.json
+  output: enrich/messages/02_output_enrich_results.json
+input_requirements: []
+output_guarantees: []
+allowed_changes:
+- add qnodes
+- add qnodes
+- modify qnodes
+- add kedges
+- add kedges
+- add results
+parameters:
+  type: object
+  properties:
+    pvalue_threshold:
+      description: The cutoff p-value for enrichment.
+      type: float
+      example: 1e-7 
+      minimum: 0
+      maximum: 1
+      default: 1e-6
+    qnode_keys:
+      description: If specified, then only knodes bound to these qnodes will be examined for enrichment and combination. 
+      type: array
+      example: [n01]
+      default: []
```

Comment:
```suggestion
```
IMO the default `[]` here is counter-intuitive. The description implies what happens when this key is left out, so I don't think we need a default. We could even go as far as to insist that the array be non-empty with `minLength: 1`.